src/domains/nodo-app/env/weu-dev/terraform.tfvars

```
@@ -0,0 +1,60 @@
+prefix          = "pagopa"
+env_short       = "d"
+env             = "dev"
+domain          = "nodo"
+location        = "westeurope"
+location_short  = "weu"
+location_string = "West Europe"
+instance        = "dev"
+
+tags = {
+  CreatedBy   = "Terraform"
+  Environment = "Dev"
+  Owner       = "pagoPA"
+  Source      = "https://github.com/pagopa/pagopa-infra/tree/main/src/nodo"
+  CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
+}
+
+lock_enable = true
+
+
+terraform_remote_state_core = {
+  resource_group_name  = "io-infra-rg"
+  storage_account_name = "pagopainfraterraformdev"
+  container_name       = "azurermstate"
+  key                  = "dev.terraform.tfstate"
+}
+
+### External resources
+
+monitor_resource_group_name                 = "pagopa-d-monitor-rg"
+log_analytics_workspace_name                = "pagopa-d-law"
+log_analytics_workspace_resource_group_name = "pagopa-d-monitor-rg"
+
+external_domain          = "pagopa.it"
+dns_zone_internal_prefix = "internal.dev.platform"
+apim_dns_zone_prefix     = "dev.platform"
+
+# chart releases: https://github.com/pagopa/aks-microservice-chart-blueprint/releases
+# image tags: https://github.com/pagopa/infra-ssl-check/releases
+tls_cert_check_helm = {
+  chart_version = "1.21.0"
+  image_name    = "ghcr.io/pagopa/infra-ssl-check"
+  image_tag     = "v1.2.2@sha256:22f4b53177cc8891bf10cbd0deb39f60e1cd12877021c3048a01e7738f63e0f9"
+}
+
+nodo_user_node_pool = {
+  enabled            = true
+  name               = "nodo01"
+  vm_size            = "Standard_D8ds_v5"
```

```suggestion
  vm_size            = "Standard_B4ms"
```
Or Standard_B8ms we expected a lot of containers running h24 (cron jobs)